runtime/base/strings/string_number_conversions.cc

```diff
@@ -0,0 +1,238 @@
+// Copyright (c) 2012 The Chromium Authors. All rights reserved.
+// Copyright (c) 2015 Intel Corporation. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "xwalk/runtime/base/strings/string_number_conversions.h"
```

Comment:
Most classes like BaseCharToDigit are defined in base/strings/string_number_conversions.cc instead of base/strings/string_number_conversions.h.
If re-use it, we have to include this  .cc.